### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by salman01zp**
## Summary of changes

_Provide a detailed description of proposed changes._

-  Fix broken links
-  Remove learn section
-  Remove DKG pallets
-  Update network configuration
-  Update node commands
-  Update node version
-  Update  Faq section

Reference Issue Checklist: #6 

### Proposed area of change

_Put an `x` in the boxes that apply._

- [x] `Overview`
- [x] `dApps`
- [x] `Tangle`

### Reference issue to close (if applicable)

_Specify any issues that can be closed from these changes (e.g. Closes #233)._

- Closes

---

### Code Checklist

- [ ] I have tested that prove my changes are working as intended
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: components/NetworkConfig.tsx [hunk 1]
```diff
@@ -98,14 +98,14 @@ const NetworkInfo = () => {
       type: "Testnet",
       symbol: "tTNT",
       decimals: 18,
-      chainId: "4006",
-      rpcUrl: "https://rpc.tangle.tools",
-      wssUrl: "wss://rpc.tangle.tools",
+      chainId: "4007",
```
Comment (1xstj) [inline components/NetworkConfig.tsx:101]: Needs to be updated

File: pages/docs/node/systemd.mdx [hunk 2]
```diff
@@ -136,7 +136,7 @@ rustup target add wasm32-unknown-unknown --toolchain nightly
         We need to generate the required keys for our node. For more information on these keys, please see the [Required Keys](https://wiki.polkadot.network/docs/learn-cryptography) section.
         The keys we need to generate include the following:
 
-        - DKG key (Ecdsa)
+        - Role key (Ecdsa)
         - Aura key (Sr25519)
```
Comment (1xstj) [inline pages/docs/node/systemd.mdx:140]: We dont have aura key anymore, replace with babe key

File: pages/docs/node/troubleshooting.mdx [hunk 3]
```diff
@@ -8,18 +8,10 @@ import Callout from '/components/Callout';
 ## Troubleshooting
 
 <Callout>
-**Logs**
+### Logs
 
 If you'd like to run the node with verbose logs, you may add the following arguments during initial setup. Adjust the target for the desired logging level (debug | error | info| trace | warn):
 
-```bash
```
Comment (1xstj) [inline pages/docs/node/troubleshooting.mdx:15]: this should be replaced with alternate log commands

**Review by 1xstj — DISMISSED** [review]
Requested changes to keys/prefix

Also would be nice to mention how the testnet/mainnet addresses will be different, mainnet would be `tg...`, testnet would be `tt...`

**Review by drewstone — APPROVED** [review]
LG @thomivy please review

**Review by thomivy — APPROVED** [review]

**Comment by thomivy** [comment]
@AtelyPham We're having the cacheing issue again I believe if youre able to reset that

**Pull request merged**